Code/Framework/AzToolsFramework/Tests/EditorVertexSelectionTests.cpp

```diff
@@ -112,26 +159,25 @@ namespace UnitTest
 
     void EditorVertexSelectionFixture::RecreateVertexSelection()
     {
+        namespace aztf = AzToolsFramework;
```

Comment:
nit: I rather avoid adding possible new conventions with `aztf` and use the actual namespace, it's not that long.